### PR TITLE
looseversion: init at 1.0.3

### PR DESCRIPTION
**Pull request by pelme**
###### Description of changes

https://pypi.org/project/looseversion/ - A backwards/forwards-compatible fork of distutils.version.LooseVersion, for times when PEP-440 isn't what you need.

salt master depends on this package, when salt 3006 comes around, this package is needed: https://github.com/saltstack/salt/commit/4ecfd3d3d549e655027a36746f9cf3bc9e11c6e9

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5563,6 +5563,8 @@ self: super: with self; {
 
   loopy = callPackage ../development/python-modules/loopy { };
 
+  looseversion = callPackage ../development/python-modules/looseversion {};
```
Comment (JulienMalka) [inline pkgs/top-level/python-packages.nix:5566]:
```suggestion
  looseversion = callPackage ../development/python-modules/looseversion { };
```

**Review by JulienMalka — CHANGES_REQUESTED** [review]

**Comment by JulienMalka** [comment]
Result of `nixpkgs-review pr 214807` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.looseversion</li>
    <li>python311Packages.looseversion</li>
  </ul>
</details>

**Comment by pelme** [comment]
@JulienMalka thanks for the feedback! is there anything else I can do to make this PR get moving?

**Comment by JulienMalka** [comment]
Looks ok to me, could you rebase your branch so that your change is only 2 commits : one where you add yourself as maintainer, one where you add the package.

**Review by JulienMalka — APPROVED** [review]

**Comment by pelme** [comment]
@JulienMalka thanks! It is now rebased on current master and only 2 clean commits.

**Review by bjornfor — APPROVED** [review]

**Comment by bjornfor** [comment]
@pelme: This PR needs a rebase, to resolve a conflict.

**Comment by pelme** [comment]
done

**Comment by pelme** [comment]
I resolved the conflict via the github ui. Will have to look into it later.

**Comment by pelme** [comment]
Updated now

**Pull request merged**